SFDigitalServices/sf-dahlia-web
# DAH-1828- Hotfix: enable preview flag on react

JIRA: https://sfgovdt.jira.com/browse/DAH-1828

Changes

- In `listing-detail.tsx` we added functionality to allow the parsing of the `preview=true` flag for the react version of the listing details page


Test Instructions

- Go to `/listings/a0W8H0000014LSAUA2?react=true`
- In network tab, ensure that the api call does not have the `force=true` flag
- Go to /listings/a0W8H0000014LSAUA2?react=true&preview=true`
- Ensure that the api call has the `force=true` flag

- You can also update a listing and salesforce and run the above test to see if the salesforce cache updates properly.

Open Question
- Is there a cleaner way to parse queryParams and pass them to the api call?
